skelebot/components/prime.py

```diff
@@ -23,7 +23,8 @@ def addParsers(self, subparsers):
         """
 
         helpMessage = "Generate Dockerfile and .dockerignore and build the docker image"
-        subparsers.add_parser("prime", help=helpMessage)
+        primeParser = subparsers.add_parser("prime", help=helpMessage)
+        primeParser.add_argument("-o", "--output", help="Output file name for the image file")
```

Comment:
If you leave `output` optional then it could be nice to let the user know the default name of the image file.

Reply:
If `output` is not provided it won't produce an image file. The default only applies to the internal function as `args.output` will be `None` if it is not provided.